include/raylib-nuklear.h

```diff
@@ -576,28 +582,47 @@ DrawNuklear(struct nk_context * ctx)
 NK_API void nk_raylib_input_keyboard(struct nk_context * ctx)
 {
     bool control = IsKeyDown(KEY_LEFT_CONTROL) || IsKeyDown(KEY_RIGHT_CONTROL);
+    bool command = IsKeyDown(KEY_LEFT_SUPER);
     bool shift = IsKeyDown(KEY_LEFT_SHIFT) || IsKeyDown(KEY_RIGHT_SHIFT);
+    #define NK_RAYLIB_INPUT_KEYBOARD_CHECK_NUM 16
+    struct nk_raylib_input_keyboard_check checks[NK_RAYLIB_INPUT_KEYBOARD_CHECK_NUM] = {
```

Comment:
My only real concern here is that the array is constructed every frame loop. Could likely use the ctx->userdata here, and move it to the NuklearUserData struct.